aws/table_aws_route53_vpc_association_authorization.go

```diff
@@ -0,0 +1,144 @@
+package aws
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/route53"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+func tableAwsRoute53VPCAssociationAuthorization(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_route53_vpc_association_authorization",
+		Description: "AWS Route53 VPC Association Authorization",
+		List: &plugin.ListConfig{
+			Hydrate: listVPCAssociationAuthorization,
```

Comment:
Here's the revised version of the sentence:
Shouldn't we handle the 'not found' error code for a non-existent Hosted Zone ID? The API throws a [NoSuchHostedZone](https://docs.aws.amazon.com/Route53/latest/APIReference/API_ListVPCAssociationAuthorizations.html#API_ListVPCAssociationAuthorizations_Errors) error if no hosted zone exists with the specified ID.